ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/OptimizingPlanner.java

```diff
@@ -123,11 +123,11 @@ public List<TaskDescriptor> planTasks() {
     List<PartitionEvaluator> inputPartitions = Lists.newArrayList();
```

Comment:
```
    double maxInputSize = MAX_INPUT_FILE_SIZE_PER_THREAD * availableCore;
```
`availableCore` should not be zero

Reply:
Okay, I think setting the minimum value of availableCore to 1 is reasonable.

Reply:
nerver mind, 0 `maxInputSize` doesn't really affect anything for planing